### ISSUE TITLE
Update to J4.1.1 Accessibility menu too wide and would need to change from 25vw to 15vw

### ISSUE THREAD

**Issue by WM-Loose** [joomla/joomla-cms#37407]

**Comment by richard67** [comment]
Translating the title:

After update to 4.1.1, the accessibility menu is too wide and should be changed from 25vw to 15vw.

**Comment by richard67** [comment]
![F24AF135-28F2-4BF6-A85B-C2F437C803D3](https://user-images.githubusercontent.com/7413183/160681777-c737f53a-21c1-474e-8e3f-4acbb6763b85.png)

(Provided by the author of this issue in other communication channel)

**Comment by Kostelano** [comment]
On a fresh install of 4.1.1, there is no such problem. Moreover, 25 units (as it really is) looks many times better than the 15 recommended by the author.

P.S. The screen of my monitor is small, screen resolution is 1366 * 768 px.

**Comment by WM-Loose** [comment]
So I think it's really broad. It's responsive, but on my 27 and 32 it looks really oversized. Wasn't that wide before?
I have now adjusted it in the user.css and custom.css.

**Comment by drmenzelit** [comment]
The width 25vw was already in 4.1, it doesn't changed with 4.1.1

**Comment by WM-Loose** [comment]
I will then close the ticket

**Comment by richard67** [comment]
So close as expected behavior?

**Comment by WM-Loose** [comment]
That's right, my customers also complain and an override isn't really a solution for me.

**Comment by WM-Loose** [comment]
Viviana is right. It was also so wide before. Therefore no error by update. Since an adjustment as an override solves the problem, I cancel the error message.
